components/ac_log/android/src/main/java/mozilla/appservices/rustlog/RustLogAdapter.kt

```diff
@@ -49,7 +51,10 @@ class RustLogAdapter private constructor(
             if (isEnabled) {
                 throw LogAdapterCannotEnable("Adapter is already enabled")
             }
+            // Tell JNA to reuse the callback thread.
+            val initializer = CallbackThreadInitializer(true, false)
```

Comment:
Let's use the three argument form so that we can have pleasant thread names.  Can we have
```
...Initializer(
    /* daemon */ true, // Don't block JVM shutdown waiting for this thread to exit.
    /* detach */ false, // Don't detach the JVM from this thread after invoking the callback.
    /* name */ "RustLogThread")
```